api/novelist.py

```diff
@@ -19,17 +19,21 @@
 )
 from core.model import (
     DataSource,
+    ExternalIntegration,
     Hyperlink,
     Identifier,
     Measurement,
     Representation,
+    Session,
     Subject,
 )
 from core.util import TitleProcessor
 
 class NoveListAPI(object):
 
     IS_CONFIGURED = None
+    __library = None
```

Comment:
I don't see that you store a `Library` here, which means I'm not sure what you do with this in `is_configured` below. If you do need to keep track of which library is associated with these credentials (and I don't think you do), store the library ID and look it up when necessary. If you store the model object itself we'll run into problems with the scoped session.